app/Main.hs

```diff
@@ -1,4 +1,3 @@
-{-# LANGUAGE OverloadedStrings #-}
```

Comment:
How do you figure out that some are not in use? Just by knowing? hlint isn't telling you, right?

Reply:
hlint told me.  But also, this file is tiny and it's easy to see there is no string literal, let alone an overloaded one :)